tool/bots.sh

```diff
@@ -152,11 +152,6 @@ $ dart run integration_test/run_tests.dart --headless"
     # TODO(https://github.com/flutter/devtools/issues/1987): rewrite integration tests.
     dart run integration_test/run_tests.dart --headless
 
-else
```

Comment:
Treat `BOT` as an optional input, if no bot configuration is provided, only run the shared part of `bots.sh`.